utils/measure/measure/powermeter/ocr.py

```diff
@@ -16,7 +17,11 @@ def __init__(self) -> None:
         self.file = open(filepath, "rb")  # noqa: SIM115
         super().__init__()
 
-    def get_power(self) -> PowerMeasurementResult:
+    def get_power(self, include_voltage: bool = False) -> PowerMeasurementResult:
+        """Get a new power reading via OCR."""
+        if include_voltage:
```

Comment:
We can remove all these checks now, because the `include_voltage=True` would only ever be called when `has_voltage_support` returns True, that's checked upfront now.

Reply:
You're sure? Those checks don't weight much and make sure that errors are caught instead of maybe returning in weird states...

Reply:
I'm doubting about it. Let's leave the exception in.